app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Api::V1::CommentsController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:create], if: -> { request.format.json? }
+
+  def index
+    user = User.find(params['user_id'])
+    post = user.posts.find(params['post_id'])
+    comments = post.comments
+    render json: comments
+  rescue StandardError => e
+    render json: { error: e.message }, status: :bad_request
+  end
+
+  def create
+    token = request.headers['X-Token']
+    user = User.find_by(api_token: token)
+    post = Post.find(params['post_id'])
+
+    puts 'Comment'
+    puts 'Comment'
+    puts 'Comment'
+    puts params
+
+    new_comment = post.comments.new(
+      text: params['text'],
+      user:
+    )
+
+    if new_comment.save
+      render json: { success: 'Comment added successfully' }, status: :created
+    else
+      render json: { error: new_comment.errors.full_messages }, status: :bad_request
+    end
+  rescue StandardError => e
+    render json: { error: e.message }, status: :bad_request
+  end
```

Comment:
- [ ] As per the project requirement, an API endpoint needs to be created to add comments to a post. However, when a request is posted to the comments endpoint with an authorization header, it returns an unauthorized request response. Therefore, it is necessary to fix this issue so that users can create comments using the API endpoints.

  ![image](https://github.com/smunoz1988/blog-on-rails/assets/73854999/f957543d-47eb-46f0-9dce-00092e8347f3)

Reply:
@Lizdev-05  Thanks for the review. We fix it using jwt gem

Comment created on postman:

![image](https://github.com/smunoz1988/blog-on-rails/assets/64698693/a203d5dc-477b-429b-aba1-fdbe7aa2c53e)

Cooment created on browser after postman Post:

![image](https://github.com/smunoz1988/blog-on-rails/assets/64698693/862f9a7a-5801-41c4-a01d-01c120b4e0cb)